dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -195,6 +196,9 @@ public String getVerifiedSource() {
         }
     }
 
+    @JsonIgnore
```

Comment:
Javadoc this (looks like a date for the underlying content, whether descriptor files or docker image) and